scripts/run_integration_tests.sh

```diff
@@ -5,23 +5,48 @@ function get_frequency_pid () {
 }
 
 function cleanup () {
-    if ${SHOULD_KILL}
-    then
-        ${RUNDIR}/kill_freq.sh
-    fi
+    local signal="$1"
+
+    case "$signal" in
+        TERM|INT) 
+            # Catch TERM and INT signals and exit gracefully
+            exit
+            ;;
```

Comment:
Seems superflous to bother catching TERM & INT if we're not going to do anything. In that case, might as well just not have a signal handler.

BUT... the point of the signal handler is to make sure that no matter how the script exits, it kills the running frequency node if it's supposed to.

If the main issue for this fix is that the script was sometimes not in the correct directory to find the path to the `kill_freq.sh` script, I think there are 2 main ways to resolve:


1. Get the directory of the running script at the top of the script file, and use that to construct the path to the `kill_freq.sh` script, instead of using relative paths.
-or-
2. Only save the PID for frequency if we start it ourselves; then in the cleanup handler, if PID is non-empty, kill it. No need for the `kill_freq.sh` script.